.cirrus.yml

```diff
@@ -34,6 +34,11 @@ gce_instance:
 
 build_task:
   alias: "build"
+  # Compiling is very CPU intensive, make it chooch quicker for this task only
+  gce_instance:
+    <<: *standard_vm
+    cpu: 8
+    memory: "16Gb"
```

Comment:
I do not think we need the extra memory, see https://cirrus-ci.com/task/6358768268083200. It was at 1.57 GB max for this task.

Reply:
Okay, so it took 01:56 like this.  I dropped it back to 4gig, let's see if the duration changes at all.  Less memory is slightly cheaper if we don't need it.  It's the CPUs that are the big expense...IIUC we're charged by the hour even if the task takes a minute.

Reply:
aaaand it looks like 8gig is the minimum if we request 8 CPUs.

Reply:
It's showing 2-minutes with 8 + 8, so let's run with that.